pgjdbc/src/main/java/org/postgresql/jdbc/PgPreparedStatement.java

```diff
@@ -1639,4 +1643,13 @@ protected void transformQueriesAndParameters() throws SQLException {
     batchStatements = newBatchStatements;
     batchParameters = newBatchParameters;
   }
+
+  //#if mvn.project.property.postgresql.jdbc.spec >= "JDBC4.2"
+  @Override
+  public long executeLargeUpdate() throws SQLException {
```

Comment:
Would you please co-locate this method with `int executeUpdate`?

Reply:
Sure.